pyuic.json

```diff
@@ -57,5 +57,5 @@
     "pyrcc": "pyside6-rcc",
     "pyrcc_options": "",
     "pyuic": "pyside6-uic",
-    "pyuic_options": "--star-imports"
-}
\ No newline at end of file
+    "pyuic_options": "--absolute-imports"
```

Comment:
users running mypy locally will need to run `python setup.py build_ui --force` once to rebuild UI files after this change